src/builtins/Transformers.jl

```diff
@@ -1022,14 +1022,58 @@ function MMI.transform(transformer::ContinuousEncoder, fitresult, X)
 end
 
 
+# # INTERACTION TRANSFORMER
+
+@with_kw_noshow mutable struct InteractionTransformer <: Static
+    order::Int                                          = 2
+    colnames::Union{Nothing, Vector{Symbol}}            = nothing
```

Comment:
I think elsewhere we generally use `features` instead of `colnames`.

Reply:
done